cmd/configschema/go.mod

```diff
@@ -489,6 +489,7 @@ require (
 	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/opencensusexporter v0.97.0 // indirect
 	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/opensearchexporter v0.97.0 // indirect
 	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/prometheusexporter v0.97.0 // indirect
+	github.com/open-telemetry/opentelemetry-collector-contrib/extension/ackextension v0.0.0-00010101000000-000000000000 // indirect
```

Comment:
Didn't 0.97 just get released? Was ackextension not part of the release? if so we need to fix that

Reply:
This is the side-effect of the addition of many replaces in the go.mod files to use the localpath ack extension to get rid of some of the CI stage failures as suggested by @crobert-1.

Reply:
We probably want to update all version numbers to `v0.97.0` for this import across the project. My understanding is that the replace below will work no matter what this version is, so it should be the latest contrib release. Let me know if you run into problems though.